src/NuGet.Clients/NuGet.CommandLine/Commands/SignCommand.cs

```diff
@@ -97,6 +97,7 @@ public SignArgs GetSignArgs()
                 CertificateStoreLocation = storeLocation,
                 CertificateSubjectName = CertificateSubjectName,
                 CertificateFingerprint = CertificateFingerprint,
+                CertificateHashAlgorithmName = HashAlgorithmName.SHA1,
```

Comment:
This is a temporary change. I will make same changes (warn for SHA-1 hash) in `nuget.exe sign` command code in a follow-up PR once this PR has been merged.

Reply:
This line of code has been removed from the latest commit since we removed the new property from the class.